portfolio/src/main/webapp/script.js

```diff
@@ -63,3 +63,35 @@ function postIt(){
 	sendEmail();
 	turn++;	
 }
+
+function loadComments(){
+	fetch('/list-comments').then(response => response.json()).then((comments)=> {
+		const commentListElement = document.getElementById('comment-list');
+		comments.forEach((comment)=>{
+			commentListElement.appendChild(createCommentElement(comment));
+		})
+	});
+}
+
+function createCommentElement(comment){
+	const commentElement = document.createElement('li');
+	commentElement.className = 'comment';
+	const titleElement = document.createElement('span');
+	titleElement.innerText = comment.title;
+	const deleteButtonElement = document.createElement('button');
+	deleteButtonElement.innerText = 'Delete';
+	deleteButtonElement.setAttribute("style", "position: relative; margin: 10px; padding: 10px; background:none ;height: 38px; font-family:'Montserrat', sans-serif; outline: none; border:none; color: #EEABA2;");
```

Comment:
Just wondering on the possibility of separating the `style` attributes into the css file. Like you have a predefined `class` or `id` and then that gets applied to this element.

Reply:
I think there is a way but I guess it does not work with chrome. I did try it that way earlier but it did not show the changes. Now I am not sure if it was the browser error or some error in my code xD

Reply:
I am thinking the browser cached the `css` file. I am not super sure either!